azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -44,13 +44,14 @@ def __init__(self, iothub_pipeline):
         self._edge_pipeline = None
 
     @classmethod
-    def create_from_connection_string(cls, connection_string, ca_cert=None):
+    def create_from_connection_string(cls, connection_string, ca_cert=None, **kwargs):
```

Comment:
>**kwargs [](start = 76, length = 8)

Question for the future: how are we going to document these kwargs?  We want them somewhere and this seems like the right place..

Reply:
Yes, API-wise I think we need to discuss as a whole team what we want this experience to be like.

---
In reply to: [330704305](https://github.com/Azure/azure-iot-sdk-python/pull/331#discussion_r330704305) [](ancestors = 330704305)

Reply:
After doing some reading, I'm on board with the use of **kwargs

---
In reply to: [331164870](https://github.com/Azure/azure-iot-sdk-python/pull/331#discussion_r331164870) [](ancestors = 331164870,330704305)